common/src/main/java/com/wynntils/features/user/GammabrightFeature.java

```diff
@@ -50,6 +50,11 @@ private void applyGammabright() {
         ConfigManager.saveConfig();
     }
 
+    private void toggleGammaBright() {
+        gammabrightEnabled = !gammabrightEnabled;
```

Comment:
You seem to have forgotten the toggling still present within applyGammabright itself

Reply:
ape mode